### ISSUE TITLE
Disable argument list wrapping for function calls in string template

### ISSUE THREAD

**Issue by paul-dingemans** [pinterest/ktlint#1861]
Ktlint does not report violations of the max line length for content inside a raw string literal. As of that, it should also not apply the argument-list-wrapping rule on code inside a string template in a raw string literal.

Original code:
```
val foo = someMethod(
    """
    longtext longtext longtext longtext longtext longtext longtext longtext  
    ${println("longtext longtext longtext longtext longtext longtext")}
    """.trimIndent(),
)
```
with `.editorconfig`:
```
root = true

[{*.kt,*.kts}]
max_line_length = 65
```

is formatted as:
```
val foo = someMethod(
    """
    longtext longtext longtext longtext longtext longtext longtext longtext  
    ${println(
        "longtext longtext longtext longtext longtext longtext",
    )}
    """.trimIndent(),
)
```

**Comment by paul-dingemans** [comment]
This should not be resolved in hindsight. To prevent this the argument list wrapping rule would need to check whether it is being applied in the context of a string template. But if this rule should not be applied in that context, the same would hold for all other rules as well that are depending on the max_line_length property. Also, it would be arbitrary that max_line_length would not be respected while other (wrapping) rules would be respected.
